lib/env.ts

```diff
@@ -7,7 +7,7 @@ const samlPath = '/api/oauth/saml';
 
 const apiKeys = (process.env.JACKSON_API_KEYS || '').split(',');
 
-const samlAudience = process.env.SAML_AUDIENCE;
+const samlAudience = `${process.env.SAML_AUDIENCE}`;
```

Comment:
Is this necessary?

Reply:
`samlAudience` is declared as a string type on [JacksonOption](https://github.com/boxyhq/jackson/pull/128/files#diff-390d2c8fe92b1151c3c68386a0c6bc3fa29585b356b6177a13e1322594e28103L138).

https://github.com/boxyhq/jackson/blob/main/npm/src/index.ts#L9

Reply:
Yes that should be optional, we should force string conversion here.